test/integration/integration_test_fixtures.py

```diff
@@ -154,6 +154,9 @@ def tearDown(self, caplog):
       for record in caplog.get_records(when):
         if record.levelno not in (logging.WARNING, logging.ERROR):
           continue
+        # Temporary hack until fixed upstream. Log does not need to be an error.
```

Comment:
We already have a mechanism for ignoring known messages from the test server's log here:

- https://github.com/envoyproxy/nighthawk/blob/f4de666db853b48a1f241ef2eaf263c2d0d3bac6/test/integration/nighthawk_test_server.py#L182
- https://github.com/envoyproxy/nighthawk/blob/f4de666db853b48a1f241ef2eaf263c2d0d3bac6/test/integration/nighthawk_test_server.py#L44-L83

If the test server is the source of the error, let's add it there. If it is the Nighthawk itself, can we implement a matching mechanism in the corresponding class instead?

Reply:
The problem has been fixed upstream so I opted to avoid this all together.